bson-scala/build.gradle

```diff
@@ -48,11 +48,4 @@ test {
     maxParallelForks = 1
 }
 
-jar {
-    bnd (
-        '-exportcontents': "*;-noimport:=true",
-        'Automatic-Module-Name': 'org.mongodb.scala.mongo-scala-bson',
-        'Build-Version': project.gitVersion,
-        'Import-Package': 'org.bson.*'
-    )
-}
+jar.manifest.attributes['Import-Package'] = 'org.bson.*'
```

Comment:
Any extra manifest attributes are added in this manner. If there are attributes from the root `build.gradle` that we need to override, the attributes are put in an `afterEvaluate` block.